client/webserver/site/src/js/wallets.ts

```diff
@@ -215,6 +228,87 @@ export default class WalletsPage extends BasePage {
       })
   }
 
+  // stepSend makes a request to get an estimated fee and displays the confirm
+  // send form.
+  async stepSend () {
+    const page = this.page
+    page.vSendFee.textContent = '0.00000000'
+    page.vSendDestinationAmt.textContent = '0.00000000'
+    page.vTotalSend.textContent = '0.00000000'
+    page.vSendErr.textContent = ''
+    page.vSendAddr.textContent = ''
```

Comment:
These are all set below before showing the form. No need to zero them.

The preferred pattern is to hide `vSendErr` until we have an error to shown. Otherwise you end up apparent inconsistent margins due to empty elements on the form.